scripts/cybershake/auto_submit.py

```diff
@@ -184,6 +186,17 @@ def submit_task(
     submitted_time = datetime.now().strftime(const.METADATA_TIMESTAMP_FMT)
     log_file = os.path.join(sim_dir, "ch_log", const.METADATA_LOG_FILENAME)
 
+    def submit_sl_script(script_name, **kwargs):
```

Comment:
What about doing a partial definition after the last if, instead of defining a new function?
https://docs.python.org/3/library/functools.html#functools.partial